cmd/appliance/functions/download.go

```diff
@@ -0,0 +1,212 @@
+package functions
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/url"
+	"os"
+	"path/filepath"
+	"strings"
+	"sync"
+
+	"github.com/appgate/sdp-api-client-go/api/v19/openapi"
+	apipkg "github.com/appgate/sdpctl/pkg/api"
+	appliancepkg "github.com/appgate/sdpctl/pkg/appliance"
+	"github.com/appgate/sdpctl/pkg/docs"
+	"github.com/appgate/sdpctl/pkg/factory"
+	"github.com/appgate/sdpctl/pkg/filesystem"
+	"github.com/appgate/sdpctl/pkg/tui"
+	"github.com/appgate/sdpctl/pkg/util"
+	"github.com/hashicorp/go-multierror"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/vbauerster/mpb/v7"
+)
+
+var (
+	logServerImages map[string]string = map[string]string{
+		"cz-opensearch":           "latest",
+		"cz-opensearchdashboards": "latest",
+	}
```

Comment:
Does env `SDPCTL_DOCKER_TAG` override the tags when downloading image with this command?

Reply:
😕 
```
SDPCTL_DOCKER_TAG=VERSION1 sdpctl appliance functions download LogServer --version VERSION2
Download complete. Files saved to /home/dln/Downloads/appgate
2 errors occurred:
        * failed to fetch image manifest: https://public.ecr.aws/v2/appgatesdp/cz-opensearchdashboards/manifests/latest - 404 Not Found
        * failed to fetch image manifest: https://public.ecr.aws/v2/appgatesdp/cz-opensearch/manifests/latest - 404 Not Found


```